### DISCUSSION TITLE
Ability to edit Category and Tag titles from top-level views

### DISCUSSION THREAD

**Discussion by epheterson** [mealie-recipes/mealie#1398] [Feature Request]
**Before submitting this feature request I have**
- [ X ] Reviewed the documentation and verified that my feature request isn't already a feature.
- [ X ] Reviewed the existing feature requests and my feature has not been requested.
- [ X ] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
I've wanted to edit a tag or category a few times and have been forced to delete/re-add instead. This isn't so bad when starting a new database, but once tags are associated with many recipes, it's very destructive to remove them and cumbersome to re-add them.

For example, I have a "Pasta" tag that I'd like to re-title to "Pasta/Noodles" and have no way of doing this.

**(Optional): Suggest A Solution**
There's already a ... on each Category or Tag to [ Delete ]. Use this area to also have a "Rename" functionality that preserves all tag/category associations but uses the new title.

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by epheterson** [comment]
I just discovered this is actually possible, though not very discoverable. If you click into a tag/category to view associated recipes, then the tag/category at the top is editable.

Since this isn't very discoverable, I think we should still add the option to modify it from Tags/Categories view where they can be modified in other ways, e.g. added and deleted.

**Reply by hay-kot** [reply]
There should probably be a more intuitive way to edit them. It's not a priority right now, but you're welcome to submit a pull request to implement a more intuitive editor.
